src/sharedData/components/SharedFileIcon.js

```diff
@@ -39,13 +40,19 @@ const ICON_FILES = {
   png: 'png.png',
 };
 
-const SharedFileIcon = ({ resourceType }) => {
+const SharedFileIcon = props => {
+  const { resourceType, styleProps } = props;
```

Comment:
Oh, whatever we do to the SharedFileIcon we should probably also do to the Link icon